Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: help setup infra infra-down dev web ai build test validate health logs seed clean
+.PHONY: help setup infra infra-down dev web ai build test validate health logs db-migrate db-reset seed clean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)